build/steps-build-release-notes.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
+- name: additionalReleaseNotesFile
+  type: string
+  default: 'Canary.md'
```

Comment:
Are we going to setup canaries for Flutter?

Reply:
No. Nice catch!